SDPi_Supplement/asciidoc/listings/vol2-clause-appendix-a-mdpws-dev-29-notification.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Please remove report parts and replace with `<!-- ... -->`.

Reply:
Plus, I fear you have to come up with listings for each report type - it is not allowed to put all reports in one message (even though I like the idea - BICEPS 2.0 perhaps...)